_layouts/default.html

```diff
@@ -7,9 +7,10 @@
 			<nav class="ui secondary menu">
 				<div class="right menu">
 					<a class="large screen only item" href="{{ site.baseurl }}#cycle">Démarche</a>
-					<a class="large screen only item" href="{{ site.baseurl }}#blog">Actualités</a>
 					<a class="item" href="{{ site.baseurl }}startups">Portefeuille</a>
-					<a class="item" id="logo" href="{{ site.baseurl }}">beta.gouv.fr</a>
+					{% if page.url != "/" %}
```

Comment:
The goal here is to check if we are on the homepage ("/") or a subpage (e.g. "/startups"). site.baseurl doesn't allow me to do that.